src/dune_sexp/escape.mli

```diff
@@ -1,2 +1,7 @@
+module Utf8 : sig
```

Comment:
If these functions are from the standard library, can you move them to `Stdune.Uchar`? The goal is that we can effectively remove that once we bump our compatibility to ocaml, so if the rest of the code can look like it's written against the stdlib it's better.

Reply:
Those functions are not coming from the standard library. The functions are inspired from `uutf` with care and adapted with our use case.